prosesstask/src/test/resources/db/migration/defaultDS/1.0.0/V1.0.0_01__Baseline_prosesstask.sql

```diff
@@ -36,7 +37,8 @@ COMMENT ON COLUMN "PROSESS_TASK"."TASK_GRUPPE" IS 'angir en unik id som gruppere
 COMMENT ON COLUMN "PROSESS_TASK"."TASK_SEKVENS" IS 'angir rekkefølge på task innenfor en gruppe ';
 COMMENT ON COLUMN "PROSESS_TASK"."NESTE_KJOERING_ETTER" IS 'tasken skal ikke kjøeres før tidspunkt er passert';
 COMMENT ON COLUMN "PROSESS_TASK"."FEILEDE_FORSOEK" IS 'antall feilede forsøk';
-COMMENT ON COLUMN "PROSESS_TASK"."SISTE_KJOERING_TS" IS 'siste gang tasken ble forsøkt kjørt';
+COMMENT ON COLUMN "PROSESS_TASK"."SISTE_KJOERING_TS" IS 'siste gang tasken ble forsøkt kjørt (før kjøring)';
```

Comment:
Endre til SISTE_KJOERING_START_TS?

Reply:
Nei, blir partisjon og expand/contract migrering så beholder as-is

Reply:
👌